dev/ci/docker_linux/Dockerfile

```diff
@@ -12,7 +12,7 @@
 
 # Last manual update 2021-09-24 (changing this comment will re-build image)
 
-FROM debian@sha256:78fd65998de7a59a001d792fe2d3a6d2ea25b6f3f068e5c84881250373577414
+FROM debian@sha256:f75d8a3ac10acdaa9be6052ea5f28bcfa56015ff02298831994bd3e6d66f7e57
```

Comment:
I wonder why this sha256 doesn't match the hash from dockerhub: https://hub.docker.com/layers/debian/library/debian/bullseye-20220418-slim/images/sha256-b771c35d1e6ecf2556718ad3c0f481b4a04c1fbc133c609643acc9dd6743ead2?context=explore

Reply:
I believe the we are missing the name of the distribution. E.g. debian:bulls-eye@sha....